src/tree/namespace/merkle_tree.cairo

```diff
@@ -11,7 +11,7 @@ struct NamespaceNode {
 #[derive(Serde, Drop, Copy)]
 struct Namespace {
     version: u8,
-    id: bytes31,
+    id: bytes31, //TODO: #28
```

Comment:
Comment should be removed since the issue was closed

Reply:
Done